spec/services/onestop_id_spec.rb

```diff
@@ -31,30 +31,30 @@
     it 'must not be blank' do
       is_a_valid_onestop_id, errors = OnestopId.validate_onestop_id_string('')
       expect(is_a_valid_onestop_id).to be false
-      expect(errors).to include 'must not be blank'
+      expect(errors).to include 'must not be empty'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.